api/api.go

```diff
@@ -87,6 +87,7 @@ func (srv *Server) initAPI(addr string) {
 	handleHTTPRequest(mux, "/wallet/send", srv.walletSendHandler)
 	handleHTTPRequest(mux, "/wallet/status", srv.walletStatusHandler)
 	handleHTTPRequest(mux, "/wallet/siafunds/balance", srv.walletSiafundsBalanceHandler)
+	handleHTTPRequest(mux, "/wallet/siafunds/spend", srv.walletSiafundsSpendHandler)
```

Comment:
I think I typically call the functions `spendCoins` in the interfaces, maybe it's best to keep parity with that.

At the same time, you really aren't spending them you are sending them. There's not any sort of contract recognized by the blockchain that you'll get something in return for doing so.